plugins/BEdita/Core/src/Model/Table/ObjectTypesTable.php

```diff
@@ -195,11 +198,14 @@ public function get($primaryKey, $options = [])
                     ->cache('map', self::CACHE_CONFIG)
                     ->toArray()
             );
-            $allTypes += array_flip(
-                $this->find('list', ['valueField' => 'singular'])
-                    ->cache('map_singular', self::CACHE_CONFIG)
-                    ->toArray()
-            );
+            if (empty($options['denySingular'])) {
```

Comment:
Does this behave nicely with cache? 🤔 I'd say it does, but I'm asking just to be sure.